app/src/main/res/values/colors.xml

```diff
@@ -4,6 +4,7 @@
     <color name="colorPrimaryDark">#009688</color>
     <color name="colorAccent">#FF4081</color>
     <color name="color_white">#FFFFFF</color>
+    <color name="color_black">#000000</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.color_black` appears to be unused
 Unused resources make applications larger and slow down builds.